src/storage.rs

```diff
@@ -418,10 +437,8 @@ pub mod test {
         let stream_name = StreamName::new_default(project_id.clone(), dataset_id.clone(), table_id.clone());
         let trace_id = "test_client".to_string();
 
-        let mut streaming = client
-            .storage_mut()
-            .append_rows(&stream_name, &table_descriptor, &[actor1, actor2], trace_id)
-            .await?;
+        let (rows, _) = StorageApi::create_rows(&table_descriptor, &[actor1, actor2]);
```

Comment:
2 comments:
- Update this test to show how to use correctly this method (i.e. retry to flush potential leftovers).
- Add another test to check the behavior when there are leftovers.

Reply:
I've factored out the retrying code in a separate function and called it twice. Once in which the rows are processed in a single AppendRows call and another time when multiple AppendRows calls are needed. Hope that this will serve as an example of how to write a typical retrying logic in case not all rows are processed in the last call.